### PR TITLE
bug in plotting function

### PR DESCRIPTION
**Pull request by stefdoerr**
@j3mdamas you can pretty much auto-accept this one. There is no `.data` in `MetricData`. It was a mistake in copy pasting some code from `Model`

### CODE REVIEW

**Comment by stefdoerr** [comment]
Well okay it ended up growing a bit more :D Still it's some necessary bug fixes with a few improvements included

File: htmd/model.py [hunk 1]
```diff
@@ -849,7 +849,11 @@ def plotFES(self, dimX, dimY, temperature, states=False, s=10, cmap=None, plot=T
         s : float
             Marker size for states.
         cmap :
-            Matplotlib colormap.
+            Control both countscmap and statecmap
```
Comment (j3mdamas) [inline htmd/model.py]: I guess it just needs a better doc? `Sets the Matplotlib colormap for both `fescmap` and `statescmap``?

Comment (stefdoerr) [reply htmd/model.py]: ok

Comment (stefdoerr) [reply htmd/model.py]: Writing an example as well

**Pull request merged**